src/common/StringUtils.h

```diff
@@ -14,18 +14,77 @@
 
 #pragma once
 
+// The functions that are here, and not in mq/base/String.h depend on third
+// party libraries.
```

Comment:
Not sure if this was intentional, but src/common/StringUtils.h by convention is a private header file

Reply:
Unintentional.  I'll evaluate.

Reply:
Instead of putting these here, I put them in mq/libutils/glmUtils.h